web/ui/src/Health/serviceHealth.js

```diff
@@ -29,62 +29,49 @@
     function($translate){
 
         var statuses = {};
-        var serviceHealths = {};
 
         // updates health check data for all services
         function update(serviceList) {
 
-            var serviceHealthCheck, instanceHealthCheck,
-                serviceStatus, instanceStatus, instanceUniqueId,
-                instance, service;
+            var serviceStatus, instanceStatus, instanceUniqueId, service;
 
             statuses = {};
 
             // iterate services healthchecks
             for(var serviceId in serviceList){
                 service = serviceList[serviceId];
-                serviceHealthCheck = serviceHealths[serviceId];
                 serviceStatus = new Status(
                     serviceId,
                     service.name,
                     service.model.DesiredState);
 
```

Comment:
remove the idea of a registry of serviceHealths since each instance now holds onto its own healthChecks